pkg/referrerstore/oras/authprovider/aws/EcrAuthTypes.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright The Ratify Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package aws
+
+import (
+	"encoding/base64"
+	"fmt"
+	"github.com/aws/aws-sdk-go-v2/service/ecr/types"
+	"strings"
+	"time"
+)
+
+// EcrAuthToken provides helper functions for ECR auth token data
+type EcrAuthToken struct {
```

Comment:
a minor comment, should this file be lower case to be consistent with other files?

Reply:
lower-cased the file